parse_parameters_test.go

```diff
@@ -0,0 +1,148 @@
+package overflow
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hexops/autogold"
+	"github.com/onflow/cadence"
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseArguments(t *testing.T) {
+	o, err := OverflowTesting()
+	require.NoError(t, err)
+	require.NotNil(t, o)
+
+	type Args struct {
+		inputArgs map[string]interface{}
+		code      string
+	}
+
+	type TestInput struct {
+		want    autogold.Value
+		want1   autogold.Value
+		wantErr autogold.Value
+		name    string
+		args    Args
+	}
+
+	tests := []TestInput{
+		{
+			name: "64bit integers",
+			args: Args{
+				code:      "u:UInt64, i:Int64",
+				inputArgs: map[string]interface{}{"u": 42, "i": 42},
```

Comment:
would it be interesting to define behavior (error?) if provided arg does not fit type? e.g. a test with `inputArgs: map[string]interface{}{"u": -42, "i": -42}`

Reply:
see the above comment

Reply:
should be underflow test instead if at all so nvm